llarp/link/iwp_internal.hpp

```diff
@@ -17,6 +18,117 @@ namespace llarp
   {
     struct LinkLayer;
 
+    using FlowID_t = llarp::AlignedBuffer< 32 >;
```

Comment:
I would do `struct FlowID_t : public AlignedBuffer< 32 > {};` here, it helps prevent implicit conversions between other typedefs to the same size `AlignedBuffer`.

Reply:
it's actually supposed to be a 16 byte thing according to spec so this change is out dated.